custom_components/tuya_local/devices/immax_neo_light_vento.yaml

```diff
@@ -0,0 +1,80 @@
+name: Immax NEO LITE VENTO Smart
+products:
+  - id: ea6vpac5hfe5rqw4
+    name: Immax NEO LITE VENTO Smart
+primary_entity:
+  entity: fan
+  dps:
+    - id: 1
+      type: boolean
+      name: switch
+      mapping:
+        - dps_val: true
+          value: ON
+        - dps_val: false
+          value: OFF
+    - id: 2
+      type: string
+      name: preset_mode
+      category: config
+      mapping:
+        - dps_val: normal
+          value: Normal
+        - dps_val: sleep
+          value: Sleep
+        - dps_val: nature
+          value: Natural
+    - id: 3
+      name: speed
+      type: integer
+      category: config
+      range:
+        min: 0
+        max: 6
+      mapping:
+        - scale: 0.06
+          constraint: preset_mode
+          conditions:
+            - dps_val: nature
+              step: 5
+            - dps_val: sleep
+              step: 5
+    - id: 8
+      name: direction
+      type: string
+      category: config
+      mapping:
+        - dps_val: forward
+          value: forward
+        - dps_val: reverse
+          value: reverse
+secondary_entities:
+  - entity: light
+    dps:
+      - id: 15
+        type: boolean
+        name: switch
+        mapping:
```

Comment:
This mapping is unneccesary

Reply:
Since the things I pointed out are not problems, I will merge this without waiting for modification.  The comments are just for your future reference.